Lib/tkinter/__init__.py

```diff
@@ -764,14 +764,32 @@ def after_cancel(self, id):
         Identifier returned by after or after_idle must be
         given as first parameter."""
         try:
-            data = self.tk.call('after', 'info', id)
+            data = self.after_info(id)
             # In Tk 8.3, splitlist returns: (script, type)
             # In Tk 8.4, splitlist may return (script, type) or (script,)
             script = self.tk.splitlist(data)[0]
             self.deletecommand(script)
         except TclError:
             pass
         self.tk.call('after', 'cancel', id)
+
+    def after_info(self, id=None):
+        """Return information about existing event handlers.
+
+        With no argument, return a list of the identifiers for all existing
+        event handlers created by the after command for this interpreter.
+        If id is supplied, it specifies an existing handler; id must have been
+        the return value from some previous call to after or after_idle and it
+        must not have triggered yet or been canceled. If the id doesn't exist,
+        a TclError is raised.  Othewise, the return value is a tuple
+        containing (script, type) where type is either 'idle' or 'timer' to
```

Comment:
The meaning of "script" is not expanded.

Reply:
Done